nextcloud_install_production.sh

```diff
@@ -157,51 +157,49 @@ You could still choose to only run on one disk though, which is not recommended,
 
-whiptail --title "Choose disk format" --radiolist --separate-output "How would you like to configure your disks?\nSelect by pressing the spacebar and ENTER" "$WT_HEIGHT" "$WT_WIDTH" 4 \
-"2 Disks Auto" "(Automatically configured)            " on \
-"2 Disks Manual" "(Choose by yourself)            " off \
-"1 Disk" "(Only use one disk /mnt/ncdata - NO ZFS!)              " off 2>results
-
-choice=$(< results)
-    case "$choice" in
-        "2 Disks Auto")
-            run_static_script format-sdb
-        ;;
-        "2 Disks Manual")
-            run_static_script format-chosen
-        ;;
-        "1 Disk")
-            print_text_in_color "$IRed" "1 Disk setup chosen."
-            sleep 2
-        ;;
-        *)
-        ;;
-    esac
+choice=$(whiptail --title "Choose disk format" --radiolist "How would you like to configure your disks?\nSelect by pressing the spacebar and ENTER" "$WT_HEIGHT" "$WT_WIDTH" 4 \
+"2 Disks Auto" "(Automatically configured)" ON \
+"2 Disks Manual" "(Choose by yourself)" OFF \
+"1 Disk" "(Only use one disk /mnt/ncdata - NO ZFS!)" OFF 3>&1 1>&2 2>&3)
+
+case "$choice" in
+    "2 Disks Auto")
+        run_static_script format-sdb
+    ;;
+    "2 Disks Manual")
+        run_static_script format-chosen
+    ;;
+    "1 Disk")
+        print_text_in_color "$IRed" "1 Disk setup chosen."
+        sleep 2
+    ;;
+    *)
+    ;;
+esac
 fi
 
 # Set DNS resolver
-whiptail --title "Set DNS Resolver" --radiolist --separate-output "Which DNS provider should this Nextcloud box use?\nSelect by pressing the spacebar and ENTER" "$WT_HEIGHT" "$WT_WIDTH" 4 \
-"Quad9" "(https://www.quad9.net/)            " on \
-"Cloudflare" "(https://www.cloudflare.com/dns/)            " off \
-"Local" "($GATEWAY + 149.112.112.112)              " off 2>results
-
-choice=$(< results)
-    case "$choice" in
-        Quad9)
-            sed -i "s|#DNS=.*|DNS=9.9.9.9 2620:fe::fe|g" /etc/systemd/resolved.conf
-            sed -i "s|#FallbackDNS=.*|FallbackDNS=149.112.112.112 2620:fe::9|g" /etc/systemd/resolved.conf
-        ;;
-        Cloudflare)
-            sed -i "s|#DNS=.*|DNS=1.1.1.1 2606:4700:4700::1111|g" /etc/systemd/resolved.conf
-            sed -i "s|#FallbackDNS=.*|FallbackDNS=1.0.0.1 2606:4700:4700::1001|g" /etc/systemd/resolved.conf
-        ;;
-        Local)
-            sed -i "s|#DNS=.*|DNS=$GATEWAY|g" /etc/systemd/resolved.conf
-            sed -i "s|#FallbackDNS=.*|FallbackDNS=149.112.112.112 2620:fe::9|g" /etc/systemd/resolved.conf
-        ;;
-        *)
-        ;;
-    esac
+choice=$(whiptail --title "Set DNS Resolver" --radiolist "Which DNS provider should this Nextcloud box use?\nSelect by pressing the spacebar and ENTER" "$WT_HEIGHT" "$WT_WIDTH" 4 \
+"Quad9" "(https://www.quad9.net/)" ON \
+"Cloudflare" "(https://www.cloudflare.com/dns/)" OFF \
+"Local" "($GATEWAY + 149.112.112.112)" OFF 3>&1 1>&2 2>&3)
+
+case "$choice" in
+    Quad9)
```

Comment:
There are no quotes here. Please fix.

Reply:
You don't need the quotes here since it is just one word without spaces.

Reply:
! @szaimen

Reply:
Okay, please wait

Reply:
The thing with a "standard" is that it should be the same everywhere.

Like
https://github.com/nextcloud/vm/pull/1071/files#diff-11ba175bd07d41f8a89a19da883fff86R21-R30
https://github.com/nextcloud/vm/pull/1071/files#diff-9fd0b2a4bd9795c3884c76936a9f11c0R113-R143
https://github.com/nextcloud/vm/pull/1071/files#diff-9fd0b2a4bd9795c3884c76936a9f11c0R173-R194

And so on... :)

Reply:
I will do that. Please be patient!

Reply:
This is done 👍